ocaml/testsuite/tests/unboxed-primitive-args/test.ml

```diff
@@ -2,18 +2,19 @@
 
 readonly_files = "common.mli common.ml test_common.c test_common.h"
 
-* setup-ocamlopt.opt-build-env
-** ocaml
+* arch_amd64
```

Comment:
I was using the `make runtest-upstream` target that gives me the `unused -msse4.2` errors for `testsuite/tests/unboxed-primitive-args`. 

I wasn't aware of the `make build_and_test_upstream` target. I will try that and let you know the result. In case the change is not needed, I will revert it.

Reply:
The above change was not required so I have reverted it.